controllers/datadogagent/feature/apm/feature.go

```diff
@@ -226,18 +222,6 @@ func (f *apmFeature) ManageDependencies(managers feature.ResourceManagers, compo
 		}
 	}
 
-	// scc
-	if f.createSCC {
-		sccName := component.GetAgentSCCName(f.owner)
```

Comment:
Should we remove the functions `GetAgentSCCName` and `GetClusterAgentSCCName` as well? They aren't used anywhere else. 
https://github.com/DataDog/datadog-operator/blob/bfe4e32720a2c71a3ef1142b3efec2df2cf896c4/controllers/datadogagent/component/utils.go#L380-L383
https://github.com/DataDog/datadog-operator/blob/bfe4e32720a2c71a3ef1142b3efec2df2cf896c4/controllers/datadogagent/component/utils.go#L351-L354